index.js

```diff
@@ -1901,6 +1901,9 @@
   //.
   //. > reduce (concat, '', Cons ('foo', Cons ('bar', Cons ('baz', Nil))))
   //. 'foobarbaz'
+  //.
+  //. > reduce (concat, '', {foo: 'ary', bar: 'Sa', baz: 'nctu'})
+  //. 'Sanctuary'
```

Comment:
I find this example more complex than necessary. I'll see whether I have permission to push an update to this branch.